src/SFA.DAS.Forecasting.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -20,6 +21,44 @@ public static class HtmlHelperExtensions
             }
 
             return new MvcHtmlString("error");
-        }   
+        }
+
+
+        public static MvcHtmlString SetZenDeskLabels(this HtmlHelper html, params string[] labels)
+        {
+            var apiCallString =
+                "<script type=\"text/javascript\">zE('webWidget', 'helpCenter:setSuggestions', { labels: [";
+
+            var first = true;
+            foreach (var label in labels)
+            {
+                if (!string.IsNullOrEmpty(label))
```

Comment:
Should we also check whitespace on this? Not sure if its a scenario we want to cover
Also I'd look at changing it to 
```suggestion
                if (string.IsNullOrWhitepsace(label))
                {
                      continue;
                }
```